src/sink/rest.rs

```diff
@@ -65,6 +65,7 @@ impl Endpoint {
         self.as_url_with_base("http://localhost")
     }
 
+    #[allow(clippy::map_err_ignore)]
```

Comment:
see above, can we include the error instead?

Reply:
in this case there is no error, only a `()`. :( shitty Url lib.